tree af5608207fc8
parent 6ff24580bca6
author Benjamin Zores <ben@geexbox.org> 1280420758 -7200
committer Benjamin Zores <ben@geexbox.org> 1280420758 -7200
revision 1492
branch default

fix MPlayer wrapper build with VDPAU when X11 support is disabled
diff --git a/src/wrapper_mplayer.c b/src/wrapper_mplayer.c
--- a/src/wrapper_mplayer.c
+++ b/src/wrapper_mplayer.c
@@ -2881,6 +2881,7 @@
       params[pp++] = "omapfb";
       break;
 
+#if USE_X11
     case PLAYER_VO_VDPAU:
     {
       int caps;
@@ -2906,6 +2907,7 @@
 
       break;
     }
+#endif
 
     case PLAYER_VO_AUTO:
     default:
